project/dexter-monitor/public/service/projectService.js

```diff
@@ -111,4 +115,62 @@ monitorApp.service('ProjectService', function($http, $log, $q) {
             });
     };
 
+    this.getCurrentStatusByGroup = function() {
+        let promises = [];
+        let statusListByProject = [];
+
+        return ServerStatusService.getActiveServerList()
+            .then((activeServerList) => {
+                statusListByProject = activeServerList;
+                statusListByProject.forEach((row) => {
+                    promises.push(setUserCount(row, row.projectName));
+                    promises.push(setDefectCount(row, row.projectName));
+                });
+
+                return $q.all(promises)
+                    .then(() => {
+                        return createStatusListByGroup();
+                    })
+                    .catch((err) => {
+                        $log.error(err);
+                        return [];
+                    });
+            })
+            .catch((err) => {
+                $log.error(err);
+                return [];
+            });
+
+
+        function createStatusListByGroup() {
+            let statusListByGroup = [];
+            const groupNameList = _.uniq(_.map(statusListByProject, 'groupName'));
+
+            groupNameList.forEach((groupName) => {
+                const groupNameFilteredStatusList = _.filter(statusListByProject, (o) => o.groupName == groupName);
+                let userCount = 0;
+                let defectCountTotal = 0;
+                let defectCountFixed = 0;
+                let defectCountDismissed = 0;
+
+                groupNameFilteredStatusList.forEach((row) => {
+                    userCount += row.userCount;
+                    defectCountTotal += row.defectCountTotal;
+                    defectCountFixed += row.defectCountFixed;
+                    defectCountDismissed += row.defectCountDismissed;
+                });
+
+                statusListByGroup.push({
+                    groupName: groupName,
+                    projectCount: groupNameFilteredStatusList.length,
+                    userCount: (_.isNaN(userCount) ? '' : userCount),
```

Comment:
Yes. When a server status is inactive, `row.userCount` in line 157 is `undefined`. So `+=` operation in line 157 makes `userCount` to `NaN`.